### ISSUE TITLE
Missing class executing Hadoop connector wordcount example

### ISSUE THREAD

**Issue by RaulGracia** [pravega/pravega-samples#107]
**Problem description**
I have build the project with `installDist` command and started `hdfs` to execute the `wordcount` example of Hadoop connector examples. However, when executing:
```~/hadoop-3.1.0/bin/hadoop jar ${HADOOP_EXAMPLES_JAR} randomtextwriter -D mapreduce.randomtextwriter.totalbytes=32000 ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}```

I got the following console output:
```2018-06-14 18:47:50,254 INFO impl.MetricsConfig: loaded properties from hadoop-metrics2.properties
2018-06-14 18:47:50,462 INFO impl.MetricsSystemImpl: Scheduled Metric snapshot period at 10 second(s).
2018-06-14 18:47:50,462 INFO impl.MetricsSystemImpl: JobTracker metrics system started
java.lang.NoClassDefFoundError: io/pravega/client/stream/Serializer
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142
```

Apparently, the application start its execution but it crashes due to a missing class `io.pravega.client.stream.Serializer.java`. I tried to add `pravega-client` and `pravega-common` in the gradle build script as dependencies and this specific problem disappears. However, the program crashes as it lacks from classes from another library (e.g., grpc). This probably requires to analyze the actual dependencies to be included in the build output jar.

**Problem location**
Gradle build script in `hadoop-connector-examples`.

**Suggestions for an improvement**
Add all the necessary libraries in the output jar.

**Comment by RaulGracia** [comment]
Found that this problem is related to #104 under the new repository organization. 
In particular, the example fails due to a wrong dependency configuration that is solved in PR #106:
```
compile     "io.pravega:pravega-connectors-hadoop:${hadoopConnectorVersion}" 
...
include dependency("io.pravega:pravega-connectors-hadoop")
```

**Comment by RaulGracia** [comment]
This problem has been solved in #104.
